### DISCUSSION TITLE
MikTeX and pdfx generate error after update

### DISCUSSION THREAD

**Discussion by nocm** [MiKTeX/miktex#875] [General]
Hi,

after an update to the latest version of MikTeX I can't generate an correct PDF/A with MikTeX, Xetex and pdfx. I'm getting the following error
`Missing \begin{document}. \includexmp{\xmp@template}`

Here is a minimal example which does not work and produced the error at my side

<pre>
\documentclass[a4paper, 12pt]{article}

\usepackage[a-1b]{pdfx}

\begin{document}
Hello, World!
\end{document}
</pre>

And the output, which will be normally generated into the file pdfa.xmpi is generated into the pdf file. I attached the pdf and log file.

[Namenlos-1.log](https://github.com/MiKTeX/miktex/files/6835064/Namenlos-1.log)
[Namenlos-1.pdf](https://github.com/MiKTeX/miktex/files/6835065/Namenlos-1.pdf)

Current Version MikTeX: 21.6.28
OS: Windows 7 Professional, 64-bit, Service Pack 1, build 7601

Thanks

**Comment by Baspla** [comment]
Hi Nocm,

I updated my MikTex yesterday and have been running into the same issue since then. I could get
\documentclass{report}
\begin{document}
test
\end{document}
to compile, however it looked weird.
Other Documents that compiled before the updates now show the exact same error you describe.

Is there a way to just downgrade again? I currently really dont have the time for this, I need latex to just work.

OS: Windows 10 Pro, 64-bit, Build: 19042.1110, Version 20H2

**Reply by nocm** [reply]
Hi,

I found a workaround for the problem
[https://tex.stackexchange.com/questions/605854/error-using-pdfx-on-tex-live-2021](url)

It seem they made a bug fix in latex and this breaks pdfx.
